inscriptions/ISic003606.xml

```diff
@@ -77,11 +79,8 @@
                         <objectDesc>
                             <supportDesc>
                                 <support>
-                                    <material key="natural rock" ref="http://www.eagle-network.eu/voc/material/lod/1.html">natural rock
-                                    </material>
-                                    <objectType key="rock_face" ref="http://www.eagle-network.eu/voc/objtyp/lod/211">rock
-                                        face
-                                    </objectType>
+                                    <material key="natural rock" ref="http://www.eagle-network.eu/voc/material/lod/1.html">natural rock</material>
+                                    <objectType key="rock_face" ref="http://www.eagle-network.eu/voc/objtyp/lod/211">rock face</objectType>
```

Comment:
I think 'stone' implies some human intervention in shaping the thing, though it's not a block or a slab. 'Rock' invokes more of a natural image.

Reply:
but this is to conflate material and object type: stone is stone, whether it's been fashioned or not... the distinction is/will be relevant for palaeographic considerations - but the material is stone, and the important distinction is the type of stone. All stone is 'natural', otherwise it is cement or something else. So, I reject the distinction myself, and think we should simply put 'stone' (eagle 2) whenever the type is not identified.

Reply:
For material yes, I was talking about object type where stone and rock are very different things. So I misunderstood your first remark, never mind :-)

Reply:
Good to know we all agree! ;-)

Reply:
So, just for now, material=stone (lod/2) and object type=rock_face?